script/brave_license_helper.py

```diff
@@ -104,6 +104,7 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
         # Brave overrides to third-party code, also covered by main notice.
         os.path.join('brave', 'third_party', 'blink'),
         os.path.join('brave', 'third_party', 'libaddressinput'),
+        os.path.join('brave', 'third_party', 'tflite_support'),
```

Comment:
moving from `chromium_src/third_party/tflite_support` which access all upstream codes to build the target.
https://github.com/brave/brave-core/pull/24953#discussion_r1718914235